contracts-periphery/Makefile

```diff
@@ -0,0 +1,8 @@
+-include .env
+
+forkBlock = 14800081
+
+build :; FOUNDRY_PROFILE=hook forge build
+install :; forge install
+Test :; FOUNDRY_PROFILE=hook forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-contract WithdrawData -vv
```

Comment:
When I attempt to run this I get a weird message and no tests run 🧐 

```
$ make Test
make: `Test' is up to date.
```

Reply:
This is because there's a folder called `test`, this came up and was fixed here: https://github.com/ScopeLift/umbra-protocol/pull/338#discussion_r882234110 (though seems this PR was not built off that PR, so the change didn't carry over)